### ISSUE TITLE
Ability to customize order of tweet text, hashtags, and link

### ISSUE THREAD

**Issue by jeffpaul** [10up/autoshare-for-twitter#66]
**Is your enhancement related to a problem? Please describe.**
[As reported on Twitter](https://twitter.com/bph/status/1220017094177312770), some people prefer to have their hashtags appear after the link in a tweet.  So the request here is to able to customize the order in which the text of the tweet, the link in the tweet, and the hashtags for the tweet appear.

**Describe the solution you'd like**
This will be dependent upon UX review on best case experience to support this request.  We'll identify implementation details once that analysis has been completed.

**Designs**
- [ ] UX review for how this could be turned into a template / settings option

**Describe alternatives you've considered**
Keep as-is with hashtags appearing before the link.

**Additional context**
none

**Comment by ibiza69** [comment]
Would be nice to heve this type of configuration and that could be SAVED, and not imput it all time we have to autoshare a post, job offer or a page:

Se precisa {POST_TITLE} #ibiza #trabajoibiza #empleoibiza #trabajaribiza #trabajo #empleo #trabajar #formentera #trabajoformentera #empleoformentera #trabajarformentera 

{PERMALINK}
+ IMAGE ATTACHED

And that when we publish and/or update a post, a job offer and/or a page, pressing publishing/update button, that the twiiter autopost worked by it self, as now you have to press on the widget button to publish/republish it. Looking for more automation and intuitive.

Thank you.

**Comment by onlanka** [comment]
Yes, as @ibiza69 said, if you can add another option in your settings page to save a message format with hashtags, emojis, it'll be easy when we post a new article, we don't have to manually write those mostly used hashtags and emojis.

Then when we hit the publish button of wordpress new post, the following template will automatically be applied.

**Eg:**

{POST_TITLE}

Read more ➡️ {PERMALINK}

#HashTag1 #HashTag2 #HashTag3

--
@jeffpaul also If someone needs to uninstall this plugin with all the saved data related to your plugin in the database, please make an option with warnings (If needed) to remove abandoned data completely.

**Comment by ibiza69** [comment]
@onlanka I read that if we need to unistall the plugin at any time, we have to make an option with warnings, and we do nto know how to do this. Can you share how to as I think the plugin should include it, but if not as you say, how could we manage to have this done correctly? Thank you.

**Comment by onlanka** [comment]
@ibiza69 Currently this plugin doesn't clean plugin related data when uninstalling the plugin, so If someone needs to uninstall + plugin data, they can check that option (it'll include warning because previous stats will also be deleted) and uninstall the plugin, others can do normal uninstallation by keeping number of data in the wordpress database.

**Comment by ibiza69** [comment]
@onlanka thank you, but I didn´t find that option at your plugin, don´t know what you mean with: "check that option", Could you please explain? We want to have control over our plugins installed, to know how to delete them well if we needed anytime. Thanks in advance for any instructions to follow for doing this correctly :)

**Comment by onlanka** [comment]
@ibiza69 That is a request for a new option to be added (currently no such option available).

I'm also a user of this plugin

**Comment by ibiza69** [comment]
Oh thanks @onlanka, didn´t knew you where not from plugin developers team. Then I really appreciate all the help you sahed with me all this past days at forum. Hopefully Developers share their knowledge on how we could include this warning for deleting well plugin db, if we wanted to do so. Cheers!!

**Comment by IlhamiD** [comment]
Great plugin, but we would appreciate more customization options.  :-)

**Comment by yasserstudio** [comment]
Hello, any updates on this issue/suggestion ?

**Comment by jeffpaul** [comment]
@yasserberrehail this is not on our immediate roadmap, though if someone were able to open a PR to address this we'd happily get it through our teams code review and testing.

**Comment by prokulit06** [comment]
Hi is this feature already implemented? Thank you
